testing/test_likelihoods.py

```diff
@@ -181,14 +181,45 @@ def testSymmetric(self):
             pred = tf.Session().run(l.predict_density(F, F, Y), feed_dict={x: l.get_free_state(), F: F_data})
             variational_expectations = tf.Session().run(l.variational_expectations(F, F, Y),
                                                         feed_dict={x: l.get_free_state(), F: F_data})
-        self.assertTrue(np.allclose(mu, np.ones((nPoints, nClasses))/nClasses, tolerance, tolerance))
-        self.assertTrue(np.allclose(pred, np.ones((nPoints, 1))/nClasses, tolerance, tolerance))
+        expected_mu = np.exp(1./nClasses * np.log(1. - epsilon) + (1. - 1./nClasses) * np.log(epsilon / (nClasses - 1)))
```

Comment:
I think mu now is very low in this case 0.031607..., which seems very low when the f for both points are equal, I assume it is just a characteristic of the RobustMax that it is like this though?